### PR TITLE
AUTHORS: Updating authors list

### PR DESCRIPTION
**Pull request by shamisp**
## What
The author list is based of git log.

## Why ?
The list has not been updated for a while

## How ?
```git shortlog --summary --numbered --email | awk '{print $2" "$3}' | sort -u```

### CODE REVIEW

**Comment by shamisp** [comment]
ok to test

**Comment by shamisp** [comment]
@yosefe I'm thinking to remove email addresses. IMHO no need.

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 10-Mar-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/b6c2227157ae28729150ae3176bb472811839d87/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/e39bf9d4f5a2ac50f1e127ebcfb411a0446d734a/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/0655fa3de1d64bf5cf7f508874cad7ba65724116/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/c16dc585e374e37d05c20de5534d60a754bf4d41/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/e321918d3febfd4e8e43794dfd83d85f3badd471/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/ae9b1083b2812384ce46d407f846fb3fa4b3a067/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/bd45ccae0fad48563d32ef1a925b201c5c3d6f6f/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/9fe0fd137e23e997d82c2abd3201d62a83e097a1/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/d1fb7a12d1b137d1dffaec0fc7f12355bbc64142/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/e38af06b8917d71343abe323f06bda9b230d52b4/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/2cdd8eeeb50ffaf55ce6a8adf2d2e508b2d494c7/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/3976f56d54eea193a848c6c727d57b6d7e53999c/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/1363ffd814c4731e9506c2babfb24294facfc62d/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/d701676ad1d658b157a222449e691c45dc9e35e4/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/474ec1d3f6d3653fe5a71ba91fe4fbdf891e9e96/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/3b592eebe8525f00048c25139c76069a263a8fec/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/5242e5978ce1c8c27b26e9ea21f779af1384323f/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/a22aac088f866cc688fa73432646f57a7d09df6e/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/22d41ea193edf0140bcae717c7a9873f2f42e0a2/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/a46a2e5a0e0ec8fded99472f86e70b6edef51643/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/fb51ce13d3343c876d8a8a464930448f1ef1cc0d/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/846abeb26f7171e8e5a68b0bf852240ec89b0b03/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/663903f1b5524f894807584488cd49d6ac0b5219/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/81e165c6063a23e3ef997301713b5412273ec52e/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/9870f402d246f40bee624735aa8e5aa0/raw/054338c6971ea6c015e58cb9d44e39732b7005fa/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 10-Mar-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/af88cfc7685965644f4ea4f9746d0bae74fb99b6/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/6cbdfe4e702750a2edd3da36ab31831911f65937/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/65a878cf6594fb59d82df3153e48f5fb958ebd0b/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/569fe536a4140ecf4f06a6a492e72479b07d0bd1/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/516cebe350bb697b6415f3625b7378e6afd48b32/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/abff4fef712550a20bfce8867a7ba441e0eaf96a/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/72b5be7044ead8e4968088d645c39a4738197272/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/6ca584d4b6498c870447aeea0280be72a3ec51ae/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/04d8023995ac4da2193d9629686ef6f5e02c9d05/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/fdaf85163e36159b6dfd4f625432eae20a9cd0ce/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/74912b451c0c47a6e897d9d7a867c5ff6cf6254c/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/10e225ee6556c8d41b3eb867beb252140b10eba5/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/70bc1a66ff755df77c2d5183efc9f054489e300a/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/1fb777b826dd7d7f554c02255c009a8faa9e1b4f/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/0505667272d4a0792e95ebfe5f5890f6e6147266/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/3ee2c7a3b0478d1d76553047619d30b2c2c737b4/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/58405ca081905df9f5169c0546ba36346762d3eb/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/e8a8cf49df26b1c815a8039acac485d155b99ddf/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/5d044a5c21feca5c3a45b155026264c558b4c697/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/fe3a553735cdb2d48c730a2e54c48c19582d6515/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/34ec858ecfe7025555812fca075096fc32679844/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/f0e765cb3bc48477440ce74b01f2f6654c5ea5a1/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/aca3b20010bcd7a76dca9c8ee6ab9498585d3e9f/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/f0341c4e55fd56d383f526d8ba634a1b4534e710/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f7375f88f885953fde835ea4a19684c6/raw/63db62b70f9f8cb86ba51ea932ffd0be0cfe0785/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by yosefe** [comment]
> @yosefe I'm thinking to remove email addresses. IMHO no need.

@shamisp I think better have email address since name+last_name is not (unlike email) a globally unique identifier

**Comment by shamisp** [comment]
@yosefe , ok we can keep it for now.

**Comment by yosefe** [comment]
@shamisp so can you pls add email address from git for the new names in the files?

**Comment by shamisp** [comment]
@yosefe I will check what can be done. Generally, the issue with emaisl that these are very often outdated or people just use "gmail" blob.

**Comment by shamisp** [comment]
@yosefe Addressing your request

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 14-Mar-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/d81c8aaf1952f13b083ddc7ee97d4ff2d6357fbb/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/7327d8df920855673e1826dd87577b2de31065eb/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/afa878c1a73933129ed5deae7e0bb45f51b773fc/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/efe97fd18303469992639dc76d289ede7f11c856/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/b6369b7973d2a64a32749964beb6e363ba527f60/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/7cba4bff45d9280a5135fc103a212e26a2469e9a/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/c10a5f8fb846adacb7c16a794ad964e627f46026/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/13aef2cecdf35299a1e4c9535e06de76650c3b3d/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/29d7b70c0922cc888b847f2153498a89cb4084fd/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/67775b641b5fb78852f51313c11e56723b49db65/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/cba44d57735488a3d60f5dd1d5a590638bb3b44f/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/a4f789d0db42bb72cc13df2df50f6e2af05d42cb/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/e34268a1a8daf949f3b3501bec9ccf780a569ce5/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/f9c9e4dee861ab0861bff905ca4af8f9967e18d1/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/a622c263782a5370e89f82de133b5a36846e4be9/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/92eeab5f535eba420b8731ac86d717bd60d457cb/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/16d2a1b2ff36e276411ceb5f0fa9e6fcb0e1d8e5/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/ecc71129eb45ce9477dc63f6bf4203ef84c90f86/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/2288a9289ebff01caa67827a531dd40faf43c1a7/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/8ef9e476769cfe98cb264de9112b170f26130ac8/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/3020a3957fbd78dcea9f5c2fc50123dc2130e2be/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/59f1aba9506516c2aa0eb4be4c476a73b8bce07f/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/8149a11ec20c9f85c4849216c1850b02b3631d82/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/7943181d876b90e1c780bd6f2b41ed2ef9034288/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/50c967c24c6b4de8640b1736e80a4a23/raw/c8be76dee7a3e70823a2fc18514c3582a4cf9354/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: AUTHORS [hunk 1]
```diff
@@ -1,39 +1,61 @@
...
+Manjunath Gorentla Venkata <manjugv@gmail.com>
+Marek Schimara <Marek.Schimara@bull.net>
 Matthew Baker <bakermb@ornl.gov>
 Mike Dubman <miked@mellanox.com>
-Mikhail Brinskii <mikhailb@mellanox.com>
+Mikhail Brinskiy <mikhailb@mellanox.com>
 Nathan Hjelm <hjelmn@lanl.gov>
 Netanel Yosephian <netanelyo@mellanox.com>
-Pavel Shamis <pavel.shamis@arm.com>
+Pak Lui <5041261+paklui@users.noreply.github.com>
```
Comment (yosefe) [inline AUTHORS]: this email is weird, should be `<Pak.Lui@amd.com>`
@paklui

Comment (shamisp) [reply AUTHORS]: @paklui - is this ok to replace GitHub blob with you actual email ? @yosefe I cannot expose developers email beyond what is officially registered at GitHub. So we have two option here: get ok from Pak, or remove it all together

Comment (paklui) [reply AUTHORS]: Yes I am ok with that. Thanks.

**Pull request merged**